app/api/server/v1/emoji-custom.js

```diff
@@ -51,6 +52,22 @@ API.v1.addRoute('emoji-custom.list', { authRequired: true }, {
 	},
 });
 
+API.v1.addRoute('emoji-custom.listWithPagination', { authRequired: true }, {
```

Comment:
I didn't like the name of the endpoint :see_no_evil: and also it doesn't follow any name convention we're using.. what do you think if we change it to `emoji-custom.all` ? we could even create a standard where all `.all` endpoint are paginated (even though I think all `.list` should be)